js/popup.js

```diff
@@ -1,11 +1,49 @@
-let pupwin = document.createElement('section');
-pupwin.className='modlpup'
-pupwin.innerHTML = '<h3>Multi-post stories</h3>'
-+'<section class="prj1_tch_dtls">'
+const info = [{
+  title: 'Multi-post stories',
+  tch_dtls: ['css', 'html', 'bootstrap', 'javascript'],
+  img_src: 'images/popup_img_mob.png',
+  des_txt: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Incidunt quibusdam odit quis saepe fuga porro inventore ipsum cum. Nulla velit aspernatur nesciunt quo eum nobis. Non neque ut accusantium tempora?0',
+}];
```

Comment:
- [ ] Please store information for all the projects and update the model based on the event. The model should pop up for all the projects in the project section.